### PR TITLE
[docs][m] Attempt to organize README by DAG

### PR DESCRIPTION
**Pull request by hannelita**
@rufuspollock WDYT? The docs for Aircan became obsolete and with too much info. What about organizing them by DAG? And maybe removing everything below the [Ignore]  tag for a while

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -311,59 +361,3 @@ python examples/ckan-upload-csv.py
 
 ## Integrations
 
-### Using Google Cloud Composer
```
Comment (rufuspollock) [inline README.md:314]: This is quite a bit of useful info we are cutting. Don't we want to keep it somewhere?

**Comment by rufuspollock** [comment]
BTW why are tests failing?

**Comment by rufuspollock** [comment]
@hannelita it looked reasonable and hard to judge exactly from review so i've merged as your approach is generally good. My 2c is that naming a bit more informatively than pure dag name would be nice but what's there is not bad and we can improve.

File: README.md (hunk 1 above)
Comment (rufuspollock) [reply README.md:314]: @hannelita ^^^

Comment (hannelita) [reply README.md:314]: This section became `ckan_api_load_gcp DAG: Using Google Cloud Composer` - there is a DAG to handle GCP

**Comment by hannelita** [comment]
@rufuspollock tests were fixed on PR #81

**Pull request merged**